modules/priceFloors.js

```diff
@@ -291,6 +300,39 @@ export function updateAdUnitsForAuction(adUnits, floorData, auctionId) {
         location: utils.deepAccess(floorData, 'data.location', 'noData'),
         floorProvider: floorData.floorProvider,
         fetchStatus: _floorsConfig.fetchStatus
+      };
+
+      if (utils.deepAccess(floorData, 'floorMin')) {
+        bid.floorData.floorMin = utils.deepAccess(floorData, 'floorMin');
```

Comment:
I can remove the deepAccess, but I left the if statement in because it cause an error passing the value as undefined in another analytics adapter when floorMin does not exist.